python/ray/air/examples/horovod/horovod_tune_example.py

```diff
@@ -52,7 +52,11 @@ def train_loop_per_worker(config):
     import horovod.torch as hvd
 
     hvd.init()
-    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+    device = (
+        ray.train.torch.get_device()
+        if torch.cuda.is_available()
+        else torch.device("cpu")
+    )
```

Comment:
```suggestion
    device = ray.train.torch.get_device()
```

`ray.train.torch.get_device()` will work regardless of gpu or cpu.